Assets/PatchKit/UnityEditorExtension/Editor/UI/SafeBuildAndUploadScreen.cs

```diff
@@ -109,25 +81,58 @@ private void DrawContent()
             {
                 GUILayout.Label("Secret:");
                 GUILayout.FlexibleSpace();
-                
+
                 GUILayout.Label(
                     _appSecret,
                     EditorStyles.miniLabel,
                     GUILayout.ExpandWidth(true));
             }
             EditorGUILayout.EndHorizontal();
+            
+            EditorGUILayout.BeginHorizontal();
+            {
+                GUILayout.FlexibleSpace();
+                if (GUILayout.Button(
+                    new GUIContent("Change application", "Change application"),
+                    GUILayout.Width(120)))
+                {
+                    Dispatch(() => SwitchLinkedApp());
+                }
+            }
+            EditorGUILayout.EndHorizontal();
+            using (Style.ColorifyBackground(Color.blue))
+            {
+                var style = new GUIStyle(GUI.skin.label);
+                style.normal.textColor = Color.blue;
+           
+                if (GUILayout.Button("Need to change platform?", style, GUILayout.ExpandWidth(false)))
+                {
+                    EditorUtility.DisplayDialog(
+                        "Need to change platform?",
+                        "PatchKit application target platform is unambiguous with current active project build platform.\n\n" +

```

Comment:
I see this text used multiple times, please put it in const value somewhere in the project.

Reply:
You can also create static function that displays this dialog.